examples/25_kimi_k2_5.py

```
@@ -0,0 +1,163 @@
+)
+
+
+def deploy_kimi_k2():
+    """Deploy Kimi-K2-Instruct with custom configuration for large model loading."""
+    client = BasilicaClient()
+
+    model = "moonshotai/Kimi-K2-Instruct"
+
+    args = [
+        "serve", model,
+        "--host", "0.0.0.0",
```

Ruff (0.14.14)

[error] Possible binding to all interfaces

(S104)